### PR TITLE
Adding Azure CLI build configuration for Quota  APIs

### PR DESCRIPTION
**Pull request by rahuls-microsoft**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ x] update existing version for new feature - Adding Azure CLI configuration for publishing.
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ x] SDK of Python
      - [ ] SDK of Java
      - [x ] SDK of Js
      - [ x] SDK of Go
      - [ x] PowerShell
      - [x ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by rahuls-microsoft** [comment]
@JustinConway Please point me to readme.cli.md commands and formats. 
I tries to test the CLI, but got following error:
TypeError: __init__() got multiple values for argument 'base_url'

How do I resolve this error?
Thanks.

**Comment by rahuls-microsoft** [comment]
Please review the PR. The CLI generation Issue is fixed.
We need to release the CLI package ASAP.

Thanks.

File: specification/reservations/resource-manager/readme.md [hunk 1]
```diff
@@ -133,50 +135,15 @@ csharp:
...
-python:
-  no-namespace-folders: true
-  output-folder: $(python-sdks-folder)/reservations/azure-mgmt-reservations/azure/mgmt/reservations
-```
-``` yaml $(python) && $(python-mode) == 'create' && !$(track2)
-python:
-  basic-setup-py: true
-  output-folder: $(python-sdks-folder)/reservations/azure-mgmt-reservations
-```
+See configuration in [readme.cli.md](./readme.cli.md)
```
Comment (qiaozha) [inline specification/reservations/resource-manager/readme.md:142]: actually readme.az.md is for azure-cli readme.cli,md is meant to for all command line tools

File: specification/reservations/resource-manager/readme.az.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+These settings apply only when `--az` is specified on the command line.
+
+``` yaml $(az)
+az:
+    extensions: quota
+    namespace: azure.mgmt.reservations
+    package-name: azure-mgmt-reservations
+az-output-folder: $(azure-cli-extension-folder)/src/reservations
+python-sdk-output-folder: "$(az-output-folder)/azext_reservations/vendored_sdks/quota"
+```
```
Comment (qiaozha) [inline specification/reservations/resource-manager/readme.az.md:12]: based on the scenario that quota and reservation is functionally independent, I would suggest you to use multi-package https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/code-gen/configure-cli.md#multi-packages

**Comment by ArcturusZhang** [comment]
Assigning this to @qiaozha since I have no knowledge on azure CLI configurations...

**Review by qiaozha — APPROVED** [review]

**Comment by rahuls-microsoft** [comment]
Please let me know, if anything else is needed from myside? 
Can we merge the PR?
Thanks.

**Comment by qiaozha** [comment]
merged

**Comment by rahuls-microsoft** [comment]
What's the process for Azure Powershell  release?

**Pull request merged**